kakao/src/main/kotlin/com/agoda/kakao/Assertions.kt

```diff
@@ -771,3 +772,17 @@ interface WebAssertions {
                 .check(WebViewAssertions.webMatches(value, matcher))
     }
 }
+
+/**
+ * Provides assertions for spinner
+ */
+interface SpinnerAssertions : BaseAssertions {
+    /**
+     * Checks if the current showing label matches given text
+     *
+     * @param text Text to be matched
+     */
+    fun withText(text: String) {
```

Comment:
I think assertion should have `hasText` name, since `withText` is usually used as matcher builder function in Kakao.